packages/vscode-extension/package.json

```diff
@@ -309,8 +309,12 @@
                   }
                 }
               },
+              "packageManager": {
+                "type": "string",
+                "description": "Package manager used for the project (eg. yarn)."
```

Comment:
if this can't be enum, can we list all allowed values in the description, there are not that many